### PR TITLE
Product slice

### PR DESCRIPTION
**Pull request by schai6**
Connects to #6 

Added basic lesson and review models, api route to get all lessons, and a lesson component for displaying all lesson information.

### CODE REVIEW

File: client/components/lessons.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          return (
+            <div key={lesson.id}>
+              <h2>{lesson.name}</h2>
+              <li>Category: {lesson.category}</li>
+              <li>Description: {lesson.description}</li>
+              <li>Image: {lesson.imageUrl}</li>
+              <li>Price: {lesson.price}</li>
+            </div>
+          )
+        })}
```
Comment (jluria) [inline client/components/lessons.js:19]: Should the parens and braces be on separate lines to align with the above lines for which they are closing?

Comment (schai6) [reply client/components/lessons.js:19]: I think it matches in line with `{lessons.map(lesson => {`. That's why they are all on the same line.

Comment (jluria) [reply client/components/lessons.js:19]: I meant that the parens on line 19 is closing the return statement on line 6, and the second brace on line 19 is closing the function opened on line 4

Comment (jluria) [reply client/components/lessons.js:19]: Currently they're all in line with the statement opened on line 9

Comment (jemyishai) [reply client/components/lessons.js:19]: Everyone open to using prettify in VS Code moving forward?

Comment (jluria) [reply client/components/lessons.js:19]: Okay cool, I didn't see it. Makes sense!

**Comment by jluria** [comment]
👍

**Review by johnson-alan — APPROVED** [review]
Looks great

**Pull request merged**